src/style.css

```diff
@@ -0,0 +1,87 @@
+* {
+  margin: 0;
+  padding: 0;
```

Comment:
```suggestion
```
do not reset margin and padding for all selectors. reset only for elements which really neet it.